assets/app/views/project.html

```diff
@@ -75,7 +75,8 @@ <h2 class="service" ng-if="!displayRouteByService[serviceId]">
                 <span ng-if="service.spec.portalIP != 'None'">
                   routing traffic on {{service.spec.portalIP}}
                 </span>
-                <span ng-if="numPorts">&#8211;
+                <span ng-if="numPorts">
```

Comment:
I've also thought that if it wouldn't be enough to add css rule `& > span { display:block }` for the `.component.meta-data`  in https://github.com/openshift/origin/blob/master/assets/app/styles/_components.less#L88